apps/server/src/schemas/apiKey.schema.ts

```diff
@@ -0,0 +1,16 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument } from 'mongoose';
+import { Recipient, RecipientSchema } from './recipient.schema';
+
+export type ApiKeyDocument = HydratedDocument<ApiKey>;
+
+@Schema()
+export class ApiKey {
+  @Prop()
+  key: string;
+
+  @Prop()
+  name: string;
+}
+
+export const ApiKeySchema = SchemaFactory.createForClass(ApiKey);
```

Comment:
Suggestion with updated  `APIKey` schema according the `/apiKeys` endpoint schema(https://github.com/diconium/ethereal-pulse/discussions/12#discussioncomment-9662606) in yaml file.


```typescript
import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
import { HydratedDocument } from 'mongoose';

export type ApiKeyDocument = HydratedDocument<ApiKey>;

@Schema()
export class ApiKey {
  @Prop({ required: true })
  name: string;

  @Prop({ enum: ['full_access', 'sending_access'], default: 'sending_access' })
  permission: string;

  @Prop({ type: Date, default: Date.now })
  createdAt: Date;

  @Prop()
  domainId?: string;

  @Prop({ required: true })
  token: string;
}

export const ApiKeySchema = SchemaFactory.createForClass(ApiKey);

```